rocky/assets/js/renderNormalizerOutputOOIs.js

```diff
@@ -0,0 +1,45 @@
+
+const buttons = document.querySelectorAll(".expando-button.normalizer-list-table-row");
+
+buttons.forEach((button) => {
+  const raw_task_id = button.closest('tr').getAttribute('data-task-id');
+  const task_id = button.closest('tr').getAttribute('data-task-id').replace(/-/g, "");
+  const json_url = window.location.href + "/" + task_id;
+
+  const getJson = (url, callback) => {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
+    xhr.responseType = 'json';
+    xhr.onload = function() {
+      var status = xhr.status;
+      if (status === 200) {
+        callback(null, xhr.response);
+      } else {
+        callback(status, xhr.response);
+      }
+    };
+    xhr.send();
+  };
+
+
+  button.addEventListener("click", () => getJson(json_url, function(err, data) {
+    if (err !== null) {
+      alert('Somthing went wrong: ' + err);
+    } else {
+      if(data.length > 0) {
+        data.forEach(object => {
+          const element = document.createElement('p')
+          const url = window.location.href.replace('/tasks/normalizers', '')
+          element.innerHTML = "<a href='" + url + "/objects/detail/?ooi_id=" + object +"'>" + object + "</a>"
+
+          button.closest('tr').nextElementSibling.querySelector('#yielded-objects-'+raw_task_id).appendChild(element);
```

Comment:
What happens if I open, and then close, and then open the same row again? Im guessing we'd end up with the same yielded objects listed twice?

Reply:
Thanks for pointing this out. I had already fixed this, but had not yet pushed my changes :)